[1.19.x] improve an expression within items/index.md

srcres258 opened the pull request:
The expression "An item(s)" is invalid on grammar when it refers to the plural meaning at which it turns into "An items". To avoid the possibly vague thoughts for some readers about this, change this into "An item or multiple items", towards which readers are able to comprehend both grammatically and misunderstanding-avoidingly.

srcres258 committed:
1.19.x: correct the spelling of one word within events.md

The word "busses" is invalid since the plural form of "bus" is only
"buses", "busses" should be the third-person singular of "bus" but not the
plural form of it. Consequently correct the mistake.

Signed-off-by: src_resources <src.res.211@gmail.com>

srcres258 committed:
Merge branch '1.19.x-fix-0' into 1.19.x

srcres258 committed:
1.19.x: improve an expression within items/index.md

The expression "An item(s)" is invalid on grammar when it refers to the
plural meaning at which it turns into "An items". To avoid the possibly
vague thoughts for some readers about this, change this into "An item or
multiple items", towards which readers are able to comprehend both
grammatically and misunderstanding-avoidingly.

Signed-off-by: src_resources <src.res.211@gmail.com>

ChampionAsh5357 commented on docs/items/index.md:
```diff
@@ -26,7 +26,7 @@ Setting the properties of an item as above only works for simple items. If you w
 ## `CreativeModeTabEvent`
 
-An item can be added to a `CreativeModeTab` via `CreativeModeTabEvent$BuildContents` on the [mod event bus][modbus]. An item(s) can be added without any additional configurations via `#accept`.
+An item can be added to a `CreativeModeTab` via `CreativeModeTabEvent$BuildContents` on the [mod event bus][modbus]. An item or multiple items can be added without any additional configurations via `#accept`.
```
```suggestion
An item can be added to a `CreativeModeTab` via `CreativeModeTabEvent$BuildContents` on the [mod event bus][modbus]. Items can be added without any additional configurations via `#accept`.
```

It makes more sense to use the plural without the article. Otherwise, the sentence flows strangely.

srcres258 replied:
Applied. Thanks.

srcres258 committed:
1.19.x: implement the suggestion given by ChampionAsh5357

It makes more sense to use the plural without the article. Otherwise, the
sentence flows strangely.

Link: https://github.com/MinecraftForge/Documentation/pull/504#discussion_r1325199491
Signed-off-by: src_resources <src.res.211@gmail.com>

srcres258 reviewed (commented):
Applied.
